kivy/utils.py

```diff
@@ -424,15 +424,17 @@ def format_bytes_to_human(size, precision=2):
 
 def _get_platform():
     # On Android sys.platform returns 'linux2', so prefer to check the
-    # existence of system specific commands and files
-    if (
-            path.exists('/default.prop')
-            and path.exists('/system/bin/logcat')
-            and path.exists('/system/xbin')
-    ):
+    # existence of environ variables set during Python initialization
+    kivy_build = environ.get('KIVY_BUILD', '')
+    if kivy_build in {'android', 'ios'}:
+        return environ['KIVY_BUILD']
```

Comment:
minor: I was just wondering why not using the `kivy_build` variable since it was created already
```
return kivy_build
```

Reply:
Yes, I forgot to change that, I will adres this in another PR, thanks @AndreMiras!!

Reply:
I was being curious, but I don't think it matter that much.
Thanks again for the quick fix

Reply:
@AndreMiras, 

Done at #6854

Thanks again!!